src/pluginFacade.ts

```diff
@@ -56,31 +60,33 @@ export { BorderingLinks }
 export { Link, LinkImplementation, overrideLink }
 export { LinkLine, LinkLineImplementation, overrideLinkLine }
 export { NodeWidget }
+export { log }
```

Comment:
using `console.log(...)` or `console.warn(...)` in plugins is completely fine as it is also forwarded to the gui (through monkey patching)